client/lib/post-normalizer/rule-content-link-jetpack-carousels.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External Dependencies
+ */
+import { forEach, get } from 'lodash';
+
+export default function linkJetpackCarousels( post, dom ) {
```

Comment:
Can we add a comment right above the rule describing what it does?

Something like:
```
The linkJetpackCarousels rule modifies all of the galleries in the content that are from jetpack sites
to link you to their host websites carousels instead of the permalink for that specific image.

For example,
Before: https://perfectlyirregular.wordpress.com/2017/03/25/my-hk-family/img_1376/
After: https://perfectlyirregular.wordpress.com/2017/03/25/my-hk-family/#jp-carousel-3060

```